### PR TITLE
Feature/ch5523/referent filter

### PR DESCRIPTION
**Pull request by tony-tran**
Please merge this https://github.com/EnMarche/worker/pull/17 before

### CODE REVIEW

File: translations/messages.fr.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@
 #
 common.gender.man: Homme
 common.gender.woman: Femme
+common.gender.male: Homme
+common.gender.female: Femme
```
Comment (ottaviano) [inline translations/messages.fr.yml]: where are these keys used?

File: tests/Controller/EnMarche/ReferentControllerTest.php [hunk 2]
```diff
@@ -18,6 +18,7 @@
 /**
  * @group functional
  * @group referent
+ * @group wip
```
[the remark this answers is not in the document]
Comment (tony-tran) [reply tests/Controller/EnMarche/ReferentControllerTest.php]: Oh my bad... Thank you

File: translations/messages.fr.yml (hunk 1 above)
Comment (tony-tran) [reply translations/messages.fr.yml]: I add this to translate the Genders class

Comment (tony-tran) [reply translations/messages.fr.yml]: Ok I see, there is `const CHOICES`, thank you

File: front/style/pages/_referent.scss [hunk 3]
```diff
@@ -37,6 +37,21 @@
                 margin: 0 5px;
             }
         }
+
+        &__search {
+            &__col {
+                flex: 1;
+                margin: 0 100px 0px 0;
```
Comment (thomasbisignani) [inline front/style/pages/_referent.scss]: `margin: 0 100px 0 0;` 

or maybe just `margin-right: 100px` ? (if is not an override)

File: src/Entity/Projection/ReferentManagedUser.php [hunk 4]
```diff
@@ -173,6 +173,11 @@ class ReferentManagedUser
      */
     private $createdAt;
 
+    /**
```
Comment (thomasbisignani) [inline src/Entity/Projection/ReferentManagedUser.php]: To be consistent with this class, you can add this PHPDoc : 
`* @var string|null`

**Pull request merged**